packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -1460,6 +1461,7 @@ class StackViewport extends Viewport implements IStackViewport {
     };
 
     triggerEvent(eventTarget, Events.STACK_VIEWPORT_NEW_STACK, eventDetail);
+    this.viewportStatus = ViewportStatus.LOADING;
```

Comment:
this was making sense to me before, now it is not. Isn't loading just before setImageIdIndex (which is responsible for loadImage?)

Reply:
The issue is that loading is sometimes used as an indicator that it is already loading an image, and setting it too early can cause an exception on init to be thrown inside the trigger event.  This was a merge from master.